services/workflows-service/prisma/schema.prisma

```diff
@@ -761,3 +766,23 @@ model Counterparty {
   @@unique([projectId, correlationId])
   @@index([correlationId])
 }
+
+model BusinessReport {
```

Comment:
Rethinking the name and meaning of this table, a report can be a general report about the business for example about his onboarding data, transaction data etc..
But this table is built for a more specific case its more a list of checks/analysis we are running on the company web presence.
With that in mind, maybe the name should be something like "CompanyWebAnalysis"

Reply:
it includes "type" which should be: audit-report, ongoing-report. T1, T2 and so on .

it can also contain different reports, transactionReport and so on.

therefore it's "vague"